### PR TITLE
Migrate to Prisma 2 for Houston API, Astro UI and minor fixes for Astro CLI

### PR DESCRIPTION
**Pull request by adam2k**
## Description

Release Houston API and Astro CLI version 0.20.0

## PR Title

> Please make the PR title informative to the user experience. For example when adding a feature, say "Add metrics tab for Deployments" instead of "Bump Astro-UI to version X.Y.Z". Bugs are sometimes harder to make customer-facing because they are very detailed, but do what you can. Instead of "Fix metrics cardinality bug" (user does not know what 'cardinality' means and does not matter to them), say "Fix bug slowing down the metrics tab". The PR title will be used for the commit message when it's merged, and these commit messages are used to generate customer-facing release notes.

> If there is only one commit in the PR, when clicking the merge button, be sure to copy the PR title into the PR squash and merge option's commit message. By default with a single commit, it will use the commit message instead of the PR title.

## 🎟 Issue(s)

This is for the 0.20.0 release

## 🧪  Testing

> What are the main risks associated with this change, and how are they addressed by tests added in this PR?

We need to make sure the manual database changes go out to production.  Follow the same steps that were used while testing on AWS, Dev and eventually Staging.

> Please add helm unit testing, if applicable. The docs are regretfully sparse for helm unit testing, [here](https://github.com/astronomer/helm-unittest/blob/main/DOCUMENT.md) is what we have to work with. In general, these kind of tests can be used for specific assertions like 'when these values are set, the resource ends up looking like XYZ' but not for generalized assertions like 'for all resources, make sure the namespace is not set to the release name'.

> Please also add to the system testing [here](../bin/functional-tests), if applicable. This kind of testing allows you to connect into any live, running pod to make assertions about how they are operating. For example, one test connects to a Prometheus pod, queries the Prometheus API, and makes assertions about the Prometheus scrape targets being healthy.

## 📸 Screenshots

> Add screenshots to illustrate the changes, if applicable.

## 📋 Checklist

- [x] The PR title is informative to the user experience

### CODE REVIEW

**Comment by adam2k** [comment]
Please hold on merge until Houston API 0.20.0 finishes building.

File: charts/astronomer/values.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ images:
     pullPolicy: IfNotPresent
   houston:
     repository: astronomerinc/ap-houston-api
-    tag: main
+    tag: 0.20.0
```
Comment (vishwas-astro) [inline charts/astronomer/values.yaml:26]: @ryanahamilton @samblackk @adam2k can we update astro-ui in this PR itself? Please change title of PR accordingly.

Comment (adam2k) [reply charts/astronomer/values.yaml:26]: It's updated now.

**Review by sjmiller609 — APPROVED** [review]
Hey looks good, feel free to merge when the images are released. CI is pretty slow today for some reason. It's ok to merge even if CI is not working in this situation because the existing code base is using mutable tags anyways.

**Pull request merged**